docs/source/contributors.rst

```diff
@@ -36,6 +36,7 @@ the package wouldn't be as rich or diverse as it is today:
  * Daniel Eriksson
  * Crispian Batstone
  * Daryl Herzmann
+ * Robert Redl
```

Comment:
Does this really belong ??

Reply:
Sure does: ``If you have been excluded from the list, please add yourself and submit a pull request.``

Reply:
Sorry, hadn't noticed who was proposing this !